IpmiFeaturePkg/GenericIpmi/StandaloneMm/StandaloneMmGenericIpmi.inf

```diff
@@ -0,0 +1,59 @@
+## @file
+# Generic IPMI StandaloneMm Driver.
+#
+# @copyright
+# Copyright 2010 - 2021 Intel Corporation. <BR>
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+[Defines]
+  INF_VERSION                         = 0x00010017
+  BASE_NAME                           = StandaloneMmGenericIpmi
+  FILE_GUID                           = CE919FB2-87DE-4583-A878-CEEAF6098A35
+  MODULE_TYPE                         = MM_STANDALONE
+  PI_SPECIFICATION_VERSION            = 0x00010032
+  VERSION_STRING                      = 1.0
+  ENTRY_POINT                         = InitializeStandaloneMmGenericIpmi
+
+[Sources]
+  ../Common/IpmiHooks.h
+  ../Common/IpmiHooks.c
+  ../Common/GenericIpmi.c
+  ../Common/IpmiInitialize.c
+  ../Common/GenericIpmi.h
+  StandaloneMmGenericIpmi.c          #GenericIpmi.c+IpmiBmcInitialize.c
+
+[Packages]
+  MdePkg/MdePkg.dec
+  IpmiFeaturePkg/IpmiFeaturePkg.dec
+
+[LibraryClasses]

```

Comment:
Can you double check these libraries?

IoLib isn't included. Report StatusCodeLib also doesn't appear to be needed. TimerLib, etc.

Reply:
The IoLib is not required, but the TimerLib is actually required in [GenericIpmi/Common/IpmiInitialize.c](https://github.com/microsoft/mu_feature_ipmi/blob/main/IpmiFeaturePkg/GenericIpmi/Common/IpmiInitialize.c)